### DISCUSSION TITLE
Creating a mesh node without a geometry

### DISCUSSION THREAD

**Discussion by jermuzet** [MPh-py/MPh#75] [General]
Hello ! I'm trying to create a comsol model from zero with the python scripting interface mph and I would like to import a .nas mesh with it. Unfortunatly when I try to create a new mesh node, it seems that I need to put a geometry in argument of the function 'create'. This is a problem for me because I want my physic to be defined directly on the imported mesh.

Is there a way to create the mesh node without geometry (as it's possible directly on comsol) ?

Thank you

**Comment by john-hen** [comment]
Hi. As the [documentation of `create()`](https://mph.readthedocs.io/en/1.1/api/mph.Model.html#mph.Model.create) notes, you need to "refer to the Comsol documentation for the values of valid arguments". I don't know more than that either. So look it up in either the [Programming Reference](https://doc.comsol.com/6.0/docserver/#!/com.comsol.help.comsol/html_COMSOL_ProgrammingReferenceManual.html) or the [API documentation](https://doc.comsol.com/6.0/doc/com.comsol.help.comsol/api/). If that's too much trouble — and it often is — then create the model in the GUI with the steps you're interested in, then save it as Java code, and look up the arguments there.

**Reply by jermuzet** [reply]
Thank you for the quick reply ! The Java code is indeed very helpful !
